app/models/party.rb

```diff
@@ -0,0 +1,6 @@
+class Party < ApplicationRecord
+  validates :movie, presence: true
+  validates :start_time, presence: true
+
+  belongs_to :movie
```

Comment:
My thinking was that a single movie can have_many parties. What if 4 separate groups want to watch Zoolander? Each group has ONE movie, but that movie HAS MANY parties! Let's talk this through.

As for as_one. To have one movie, the party id would need to be in the movie table from my understanding. Let's discuss!

Reply:
Cool, I'm following you. I turned myself around last night trying to match what you built vs what I had imagined. Feels like this will work great and that we'll be able to add other associations as needed.